### PR TITLE
Issue678 window states

### LINKED ISSUE #678: Provide option for not lumping thermal mass into one state
We lump the thermal mass of the window glazing into a single state.
This is currently enforced unless you're an expert. We should provide this as a default option.

### PR DESCRIPTION
**Pull request by Mathadon**
@damienpicard @GlennReynders this closes #678

See #678 for an explanation of what I did.

### CODE REVIEW

**Comment by Mathadon** [comment]
Note that there are some large differences in reference results for the `Structure` example. These were caused by f32aa07f70a4bd57966b67766cf973cf9d660512

**Comment by damienpicard** [comment]
Fine for me, but I let @GlennReynders have his opinion and merge it.

**Comment by GlennReynders** [comment]
@Mathadon

File: IDEAS/Buildings/Components/Window.mo [hunk 1]
```diff
@@ -69,7 +69,7 @@ protected
     "Add lumped thermal capacitor for window glazing";
   final parameter Boolean addCapFra =  fraType.present and not energyDynamics == Modelica.Fluid.Types.Dynamics.SteadyState
     "Added lumped thermal capacitor for window frame";
-  final parameter Modelica.SIunits.HeatCapacity Cgla = layMul.C
+  final parameter Modelica.SIunits.HeatCapacity Cgla = layMul.C*(1-fraC)
```
Comment (Mathadon) [inline IDEAS/Buildings/Components/Window.mo]: I think the old implementation was more correct?

Comment (Mathadon) [inline IDEAS/Buildings/Components/Window.mo]: @GlennReynders is this ok?

Comment (Mathadon) [inline IDEAS/Buildings/Components/Window.mo]: pump

**Pull request merged**